modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/resources/content/Language.properties

```diff
@@ -1,42 +1,42 @@
 advanced-configuration=Advanced Configuration
 analyzer=Analyzer
-analyzer-help=The analyzer that is used to analyze the free form text. Defaults to the analyzer associated with the first field in fields.
+analyzer-help=Specify the analyzer to use on the input document's fields. If left blank, this defaults to the analyzer associated with the first entry in the fields configuration.
 document-entry=Document Entry
-document-entry-help=Represents the document entry that is displayed through this template.
+document-entry-help=Set the document entry that is displayed through this template.
 federated-search-key=Federated Search Key
 federated-search-key-help=Enter the key of an alternate Search this widget is participating on.
```

Comment:
Just to make sure, 'Search' on this line is intentionally capitalized, right?

Reply:
No, it should probably be lower case.